crates/pallet-native-to-evm-currency-swap/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "pallet-native-to-evm-currency-swap"
```

Comment:
```suggestion
name = "pallet-native-to-evm-swap"
```

We can actually drop the "currency" from the name of this pallet, as it no longer requires it to be implemented

Reply:
Got it, done - https://github.com/humanode-network/humanode/pull/1449/commits/4c9425cb193ebcace3e7bddfb39b205262ddefc2